assets/helpers/tracking/conversions.js

```diff
@@ -1,19 +1,13 @@
 // @flow
 
-import { doNotTrack } from 'helpers/page/page';
 import { getAbsoluteURL } from '../url';
 import { pageView } from './ophanComponentEventTracking';
-import { successfulConversion } from './googleTagManager';
 import type { Participations } from '../abTests/abtest';
 
 export default function trackConversion(
```

Comment:
I'd actually go as far as to say we should remove all calls to `trackConversion`. We definitely don't use that Ophan pageview to track conversions - if anything it will just add noise to our pageview data. The only thing to check would be if `trackConversion` is called in any subscriptions-related code, whether S&C team need it. But from a Contributions perspective, I think we can ditch this entire function

Reply:
@jranks123 would you agree?